src/adaptors/sdai/index.js

```diff
@@ -0,0 +1,72 @@
+const utils = require('../utils');
+const bridgeInterestReceiverABI = require('./bridgeinterestreceiver');
+const erc20ABI = require('./erc20');
+const Web3 = require('web3');
+
+const chains = {
+  xdai: {
+    sDAI: '0xaf204776c7245bF4147c2612BF6e5972Ee483701',
+    wxDAI: '0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d',
+    interestreceiver: '0x670daeaF0F1a5e336090504C68179670B5059088',
+  },
+};
+let web3;
+let bridgeInterestReceiver;
+let wxDAI;
+
+async function init() {
+  try {
+    web3 = new Web3(
```

Comment:
pls avoid using this and instead use our sdk instead, the chain key you have to use is 'xdai'